moskito-backend/src/Service/ClosestCampsitesService.php

```diff
@@ -0,0 +1,35 @@
+<?php 
+
+
+namespace App\Service;
+
+class ClosestCampsitesService {
+
+    const EARTH_RADIUS = 6371;
+
+    private array $closestCampsites = [];
+    
+    public function getClosestCampsites( array $filter, array $sortedCampsites): array {
+
+        $latFrom = deg2rad($filter['latitude']);
```

Comment:
Nicely implemented. However, abbreviations lat/lng should be avoided here (...as I was told).

Reply:
why?

Reply:
I got this feedback for my identical code. (But to be honest I didn't change it either.)

Reply:
will go for that as well...